mods/beds/spawns.lua

```diff
@@ -41,10 +41,12 @@ function beds.save_spawns()
 	if not beds.spawn then
 		return
 	end
+	local data = {}
 	local output = io.open(org_file, "w")
 	for i, v in pairs(beds.spawn) do
```

Comment:
Right...  Glad you cleared that up, but for the sake of a variable name I'll leave it as is so long as things are working.